CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## UNRELEASED
 
+- Clarified the use of SVGs with a new test and changes to documentation. 
```

Comment:
This can be moved under the other unreleased heading. We will change the 'UNRELEASED' to a version number when we publish all the changes.